contracts/test/contract.SL2RD.test.js

```diff
@@ -607,13 +607,45 @@ contract("SL2RD", (accounts) => {
 
       // Try and initiate distribution with non-operator && non-owner address
       try {
-        await nonOperatorAddress.transferNextAvailable(
-          recipientAddress
-        );
+        await splitContract.transferNextAvailable(recipientAddress, {
+          from: nonOperatorAddress,
+        });
         assert(error.message.includes("SHARE030"));
       } catch (error) {
         console.log(error.message);
       }
     }
   );
+
+  specify(
+    "Initialize operator registry and use verified operator address to conduct transfer.",
+    async () => {
+      const shareContract = await SHARE.deployed();
+      const splitContract = await SL2RD.new();
+      const operatorRegistry = await OperatorRegistry.deployed();
+      const ownerAddresses = Array(10).fill(accounts[0]);
+      const recipientAddress = accounts[NON_OWNER_ADDRESS_INDEX];
+      const uniformCollaboratorsIds = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+      const operatorAddresses = [accounts[5], accounts[6]];
+      const nonOperatorAddress = accounts[7];
+      const communitySplitsBasisPoints = 4000;
+
+      await operatorRegistry.initialize(operatorAddresses);
+
+      await splitContract.initialize(
+        ownerAddresses /* addresses_ */,
+        uniformCollaboratorsIds /* tokenIds_ */,
+        communitySplitsBasisPoints /* communitySplitsBasisPoints_ */,
+        shareContract.address /* shareContractAddress_ */,
+        operatorRegistry.address /* operatorRegistryAddress_ */
+      );
+
+      assert(
```

Comment:
@DeusExM4ch1n4 What's the assertion here?

Reply:
a plain assert() will make sure whatever happens inside of it is functional, meaning it is true or succeeds.  Specifically here we want to make sure the operation succeeds even with an operator address.